ITensorGaussianMPS/src/gmps.jl

```diff
@@ -287,21 +661,57 @@ a matrix product state (MPS).
 The correlation matrix should correspond to a pure state (have all eigenvalues
 of zero or one).
 """
-function correlation_matrix_to_mps(
+function convert_correlation_matrix(Λ::AbstractMatrix, s::Vector{<:Index})
```

Comment:
Maybe `symmetric_correlation_matrix`? `convert_*` is a bit general here (if I saw this function out of context I wouldn't know what it was converting to).

Reply:
Ok.